pre_commit/store.py

```diff
@@ -188,7 +188,8 @@ def _git_cmd(*args: str) -> None:
 
     LOCAL_RESOURCES = (
         'Cargo.toml', 'main.go', 'go.mod', 'main.rs', '.npmignore',
-        'package.json', 'pre_commit_placeholder_package.gemspec', 'setup.py',
+        'package-dev-1.rockspec', 'package.json',
```

Comment:
package-dev-1.rockspec should probably have a less generic name so it's clear it comes from `pre-commit`